packages/stateful/actions/components/Stake.tsx

```diff
@@ -285,7 +286,7 @@ export const StakeComponent: ActionComponent<StakeOptions, StakeData> = ({
 
       {/* If not claiming (i.e. withdrawing reward), show amount input. */}
       {stakeType !== StakeType.WithdrawDelegatorReward && (
-        <div className="flex flex-row gap-2">
+        <div className="flex flex-col gap-2 xs:flex-row">
```

Comment:
**thought**: not tested, but could be the same thought as above?

```suggestion
        <div className="flex flex-col gap-2">
```

Reply:
Same as above.